libs/coin-framework/src/mocks/fixtures/nfts.ts

```diff
@@ -307,6 +307,18 @@ export const NFTs = [
   },
 ];
 
+export const NFTs_STAX = [
```

Comment:
just in terms of naming the convention here seems to be `NFTs_{blockchain}` so maybe keep that and do something like `NFTs_ETHEREUM_STAX` and add a comment to explain what it is

Reply:
Done